distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1871,6 +1871,86 @@ type Table
             new_column = column_to_parse.parse type format on_problems
             table.set new_column new_name=column_to_parse.name set_mode=Set_Mode.Update
 
+    ## GROUP Standard.Base.Conversions
+       Formats `Column`s within a `Table` using a format string,
+       `Date_Time_Formatter`, or `Column` of format strings.
+
+       Arguments:
+       - columns: The columns to format. If not specified, all columns will be
+         formatted. The columns can have different types, but all columns must
+         be compatible with any provided `format` value.
```

Comment:
I don't think we should have a default on this.

In `parse`, all `Text` columns are parsed (but in Auto mode, if no more specific type is found, they just stay as-is).

With `format` it's not really reversible, because as you mention - the format string needed depends on the type - so if I understand this correctly, if the Table has columns of different types, this will always error?

IMO a default is just not needed - the usecase for `format` is that I select a few columns that I want to be formatted.

```suggestion
       - columns: The columns to format.
```

Reply:
The only case where you can format the whole table is if you use the default formatter ("" or Nothing), but since that's an unlikely case (compared to parsing a whole table), I'll take out the default.